docs/madr/decisions/049-policy-on-namespace.md

```diff
@@ -0,0 +1,529 @@
+          kuma.io/zone: zone-1
+      rules:
+        - matches:
+            - path:
+                type: PathPrefix
+                value: /v2
+          default:
+            backendRefs:
+              - kind: MeshServiceSubset
+                name: backend
+                tags:
+                  version: v2
+---
+apiVersion: kuma.io/v1alpha1
+kind: MeshTimeout
+metadata:
+  name: backend-route-timeout
+  namespace: backend-ns
+  labels:
+    kuma.io/zone: zone-1
+spec:
+  targetRef:
+    kind: MeshHTTPRoute
+    name: backend-route # name is enough as both policies are in the same namespace
+  to:
+    - targetRef:
+        kind: Mesh
+      default:
+        http:
+          requestTimeout: 5s
+---
+apiVersion: kuma.io/v1alpha1
+kind: MeshTimeout
+metadata:
+  name: frontend-to-backend
+  namespace: frontend-ns
+  labels:
+    kuma.io/zone: zone-2
+spec:
+  targetRef:
+    kind: MeshHTTPRoute
+    labels:
+      kuma.io/display-name: backend-route
+      k8s.kuma.io/namespace: backend-ns
+      kuma.io/zone: zone-2
+  to:
+    - targetRef:
+        kind: Mesh
+      default:
+        http:
+          requestTimeout: 3s
+```
+
+### MeshGateway Examples
+
+As a **Cluster Operator** I want to deploy a single global `MeshGateway` to the cluster so that I’m able to manage a
+single deployment for the whole cluster without knowing specific details about apps and routes.
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshGateway
+mesh: default
+metadata:
+  name: edge-gateway
+  labels:
+    kuma.io/origin: zone
+spec:
+  conf:
+    listeners:
+      - port: 80
+        protocol: HTTP
+  selectors:
+    - match:
+        kuma.io/service: edge-gateway
+```
+
+#### Targeting MeshGateway in MeshHttpRoute
+
+As a **Frontend Service Owner** I want to configure routes on `MeshGateway` without the access to
+cluster-scoped CRDs so that I could manage my routes independently.
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshHTTPRoute
+metadata:
+  name: http-route-1
+  namespace: frontend-ns
+  labels:
+    kuma.io/origin: zone
+spec:
+  targetRef:
+    kind: MeshGateway
+    name: edge-gateway
+  to:
+    - targetRef:
+        kind: Mesh
+      rules:
+        - matches:
+            - path:
+                type: PathPrefix
+                value: /gui
+          default:
+            backendRefs:
+              - kind: MeshService
+                name: frontend
+                namespace: frontend-ns
+```
+
+#### Targeting MeshGateway in other policies
+
+As a **Frontend Service Owner** I want to configure circuit breaker on `MeshGateway` without the access to
+cluster-scoped CRDs so that I could manage my routes independently.
+
+```yaml
+type: MeshCircuitBreaker
+mesh: default
+name: circuit-breaker
+namespace: frontend-ns
+spec:
+  targetRef:
+    kind: MeshGateway
+    name: edge-gateway
+  to:
+  - targetRef:
+      kind: MeshService # you need to specify MeshService in to[].targetRef.kind
+      name: frontend
```

Comment:
This is equivalent to having the implicit `namespace: frontend-ns` ? 
What would happen if I had `namespace: backend-ns`

Finally what happens with:

```yaml
type: MeshCircuitBreaker
mesh: default
name: circuit-breaker
namespace: frontend-ns
spec:
  targetRef:
    kind: MeshGateway
    name: edge-gateway
  to:
  - targetRef:
      kind: MeshService # you need to specify MeshService in to[].targetRef.kind
      name: frontend
     default: {}
  - targetRef:
      kind: MeshService
      name: backend
      namespace: backend-ns
    default: {}
```

My gut feeling is that this should rejected (entries in the `to` must have the same namespace). We should likely suggest users to not set multiple `to` entries